src/wayfinding/types/index.ts

```diff
@@ -2,40 +2,109 @@ import { AccessibilityMode } from "src/sdk";
 
 import type { DirectionsRequest, Point } from "../../sdk/types";
 import { ErrorName } from "./constants";
+import type { StyleProp, ViewStyle } from "react-native";
 
 export interface MapViewError {
   name: ErrorName;
   description: string;
 }
 
+export type MapViewConfiguration = {
+  /**
+   * A String parameter that allows you to choose the API you will be retrieving our cartography from. Default is "dashboard.situm.com".
+   */
+  apiDomain?: string;
+  /**
+   * A String parameter that allows you to specify which domain will be displayed inside our webview. Defaults to "https://map-viewer.situm.com/".
+   */
+  viewerDomain?: string;
+  /**
+   * @required
+   * Your Situm API key. Find your API key at your [situm profile](https://dashboard.situm.com/accounts/profile)
+   */
+  situmApiKey: string;
+  /**
+   * A String identifier that allows you to remotely configure all map settings. Alternatively you can pass a buildingIdentifier, remoteIdentifier will be prioritized.
+   */
+  remoteIdentifier?: string;
+  /**
+   * @required
+   * The building that will be loaded on the map. Alternatively you can pass a remoteIdentifier (that will be prioritized).
```

Comment:
sona raro que o parametro sexa required e na descripción poñas -> Alternatively you can pass a remoteIdentifier

Reply:
"In case you specify a remoteIdentifier, this parameter might be overwritten"
Asi esta mellor explicado non ?